paper.md

```diff
@@ -39,135 +39,125 @@ bibliography: paper.bib
+via the [xeus-cling](https://github.com/QuantStack/xeus-cling) project.
+This is demonstrated interactively on the [mlpack homepage](https://www.mlpack.org).
+Examples of C++ notebooks can be found in the
+[mlpack examples repository](https://github.com/mlpack/examples),
+and these can easily be run on BinderHub.
+
+*New bindings and enhanced availability.*
+Support for the Julia, Go, and R languages 
 [@bezanson2017julia, @pike2012go, @rcore2022, @parihar2022rmlpack]
-languages has been added via mlpack's automatic binding system.  These bindings
-can be used by installing mlpack from the language's package manager (`Pkg.jl`,
-`go get`, `install.packages('mlpack')`).  In addition, since mlpack's reduced dependency footprint has
-significantly simplified the deployment process, mlpack's Python dependencies
-are now available for numerous architectures---including Windows---both on PyPI
-and in `conda-forge`.
-
-*Cross-compilation support and build system improvements.* As mentioned earlier,
-mlpack's CMake configuration now supports easy cross-compilation, for instance
+has been added via mlpack's automatic binding system.  These bindings
+can be used by installing mlpack from the language's package manager
+(`Pkg.jl`, `go get`, `install.packages('mlpack')`).
+Furthermore, since mlpack's reduced dependency footprint has significantly
+simplified the deployment process, mlpack's Python dependencies
+are now available for numerous architectures both on PyPI and in `conda-forge`.
+
+*Cross-compilation support and build system improvements.* 
+mlpack's build configuration now supports easy cross-compilation, for instance
 via toolchains such as [buildroot](https://buildroot.org).  By specifying a few
-CMake flags, a user may produce a working mlpack setup for a variety of embedded
-systems.  This required the implementation of a dependency auto-downloader,
+flags, a user may produce a working mlpack setup for a variety of embedded systems.
+This required the implementation of a dependency auto-downloader,
 which is capable of downloading [OpenBLAS](https://github.com/xianyi/OpenBLAS)
-and compiling (if necessary) for the target architecture.  This auto-downloader
+and compiling (if necessary) for the target architecture.  The auto-downloader
 can also be enabled and used for any situation, thus easing installation and
 deployment.
 
-# Special Thanks
+# Acknowledgements
 
-mlpack is a community-led library, and the software is the product of the hard
-work of over 220 individuals (at the time of this writing).  We are indebted to
-all of these contributors, as well as all those who have opened issues and bug
-reports over the years.  mlpack's development has been supported by Google, via
-a decade-long participation the Google Summer of Code program, and also by
-NumFOCUS, which fiscally sponsors mlpack.
+Development of mlpack is a community effort. It is the product of hard work by
```

Comment:
```suggestion
Development of mlpack is community-led. It is the product of hard work by
```
I'll make this change to highlight that we are actually, really, truly community-led, not guided by corporate overlords like many other machine learning libraries.  (Not as a knock on those libraries, but I think it is important to differentiate.)